index.html

```diff
@@ -63,7 +63,7 @@ <h3>User Profile</h3>
         </td>
       </tr>
     </table>
-    <script>
+    <script type="text/javascript" src="dist/bundle.js">
```

Comment:
Also here, is there a specific reason you are bringing in the app bundle as this index.html is not what is getting served up by webpack?